### DISCUSSION TITLE
How to download Share with curl / wget cli tools.

### DISCUSSION THREAD

**Discussion by tmatejuk** [drakkan/sftpgo#1399] [General]
Hi.

I use self hosted SFTPGo version 2.5.4-cc381443 .

My question is about Share download possibility with curl / wget cli tool.

Let's say I already have (in SFTPGo) directory with two ISO files ( directory name : test_iso ).

I've created Share : 
name : 'test_iso_share'
scope: Read
path :  /test_iso 
password : PassWord 

Let's say 'Share access link' is https://example.com/web/client/pubshares/aaaBBBcccDDDeee

Is it possible, how could I use curl / wget cli tool to download this Share ?

**Comment by drakkan** [comment]
Hello,

this can be done using the REST API, check the [docs](https://sftpgo.stoplight.io/docs/sftpgo)
